src/index.html

```diff
@@ -10,7 +10,40 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page">
+    <form
+      action="#"
+      method="post"
+      class="big_form"
+      data-qa="big"
+    >
+      <img src="images/Search.svg" alt="Search"
+        class="big_icon"
+      >
```

Comment:
```suggestion
      <img
        src="images/Search.svg"
        alt="Search"
        class="big_icon"
      >
```
fix your `<img>` tags style 😉